lib/pluginmanager/install.rb

```diff
@@ -44,9 +44,11 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    options = { :rubygems_source => gemfile.gemset.sources }
```

Comment:
This part does not mess with sources :+1: , that it does is pulling out the sources from the gemfile and let later know rubygems. If I do understand you right, you mean I should do what? pass the gemfile to the bundler class? can you elaborate a bit more please?